IntroOOP/BankAccount/BankAccount.cs

```diff
@@ -150,4 +144,34 @@ public bool TransferMoney(BankAccount source, decimal amount)
     /// Метод печатает состояние
     /// </summary>
     public void GetInfo() { Console.WriteLine("Номер счета: {0}\nБаланс: {1:N} RUB\nТип: {2}\n", ID, Balance, Type); }
+
+
+    public static bool operator ==(BankAccount a, BankAccount b) => a.Equals(b);
```

Comment:
Если в переменной `a` будет `null`, то программа упадёт.
```suggestion
    public static bool operator ==(BankAccount a, BankAccount b) => Equals(a, b);
```